flytekit/core/interface.py

```diff
@@ -7,7 +7,7 @@
 from collections import OrderedDict
 from typing import Any, Dict, Generator, List, Optional, Tuple, Type, TypeVar, Union
 
-from typing_extensions import get_args, get_origin
+from typing_extensions import get_args, get_origin, get_type_hints
```

Comment:
so in python 3.7 and 3.8, it just doesn't do anything?

Reply:
no, the annotations are returned regardless in python < 3.9.

Reply:
I guess that's what you meant by "doing nothing".